src/v2/Apps/ViewingRoom/__tests__/ViewingRoomClosedApp.jest.tsx

```diff
@@ -0,0 +1,98 @@
+import React from "react"
+import { MockBoot, renderRelayTree } from "v2/DevTools"
+import ViewingRoomApp from "../ViewingRoomApp"
+import { graphql } from "react-relay"
+import { ViewingRoomClosedApp_Test_QueryRawResponse } from "v2/__generated__/ViewingRoomClosedApp_Test_Query.graphql"
+import { Breakpoint } from "@artsy/palette"
+
+jest.unmock("react-relay")
+jest.mock("v2/Artsy/Router/useRouter", () => ({
+  useRouter: () => ({
+    match: {
+      params: {
+        slug: "subscription-demo-gg-guy-yanai",
+      },
+    },
+  }),
+  useIsRouteActive: () => false,
+}))
+
+describe("ViewingRoomApp", () => {
+  const slug = "subscription-demo-gg-guy-yanai"
+
+  const getWrapper = async (
+    breakpoint: Breakpoint = "lg",
+    response: ViewingRoomClosedApp_Test_QueryRawResponse = ViewingRoomClosedAppFixture
+  ) => {
+    return await renderRelayTree({
+      Component: ({ viewingRoom }) => {
+        return (
+          <MockBoot breakpoint={breakpoint}>
+            <ViewingRoomApp viewingRoom={viewingRoom}>
+              some child
+            </ViewingRoomApp>
+          </MockBoot>
+        )
+      },
+      query: graphql`
+        query ViewingRoomClosedApp_Test_Query($slug: ID!) @raw_response_type {
+          viewingRoom(id: $slug) {
+            ...ViewingRoomApp_viewingRoom
+          }
+        }
+      `,
+      variables: {
+        slug,
+      },
+      mockData: response,
+    })
+  }
+
+  it("renders the correct components", async () => {
+    const wrapper = await getWrapper()
+    expect(wrapper.find("ViewingRoomMeta").length).toBe(1)
+    expect(wrapper.find("AppContainer").length).toBe(1)
+    expect(wrapper.find("ViewingRoomHeader").length).toBe(1)
+    expect(wrapper.find("ViewingRoomTabBar").length).toBe(0)
```

Comment:
Meaning, all of this kind of stuff is tested elsewhere. What we should be checking is that the behavior specific to this addition is valid (does `<ViewingRoomClosed>` appear after a certain date time; is it hidden before that time; does it alone contain the correct internal component structure).

Reply:
This behavior is based on the value we are getting from the server. We are not planning to update the VR dynamically here once it's loaded. So if person enters the room before it is closed - he will see all the data before page reloads. So this test is just testing if one version is rendered vs the other. But I agree - it's confusing to break this spec into 2 files. I'll try to consolidate it there.

Reply:
Exactly. When the app initializes - it either renders the `Tabs` or the `Closed` version. Does that not sound right @damassi ?

Reply:
I should probably add a spec for ViewingRoomClosed component itself that just checks that it renders text and a link though:)

Reply:
This makes sense! That top-level behavior testing can all live [in here](https://github.com/artsy/force/blob/master/src/v2/Apps/ViewingRoom/__tests__/ViewingRoomApp.jest.tsx). If we need to create a new test file it can be specific to the contents of `<ViewingRoomClosed>`.